src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -521,6 +537,7 @@ def validate_relative_risk_rebin_source(
 
 
 def validate_rebin_source(builder, risk: EntityString, data: pd.DataFrame):
+    # TODO: when this is Falsey, it now throws a ConfigurationError
```

Comment:
What is this todo?

Reply:
Just a reminder for our discussion yesterday